src/org/jitsi/impl/neomedia/rtp/RTPEncodingImpl.java

```diff
@@ -279,11 +279,15 @@ public boolean matches(byte[] buf, int off, int len)
             return false;
         }
 
+        if (temporalId == -1)
+        {
+            return true;
+        }
+
         int tid = track.getMediaStreamTrackReceiver()
             .getStream().getTemporalID(buf, off, len);
 
-        return (tid == -1 && ArrayUtils.isNullOrEmpty(dependencyEncodings))
-                || tid == temporalId;
+        return tid == temporalId;
```

Comment:
I don't think anybody suggests that simulcast and TS are the same. All I'm saying is that if you send 3 independent simulcast streams, and each stream is encoded without temporal layers, then this needs to be reflected in signaling (somehow), driving the MediaStreamTrackFactory to create a MST with 3 encodings (each with TID = -1), as opposed to 9 (each with positive TID).